src/components/uielements/assets/assetIcon/AssetIcon.tsx

```diff
@@ -63,12 +62,7 @@ const AssetIcon: React.FC<Props> = (props: Props): JSX.Element => {
     )
   }, [asset, size])
 
-  return (
-    <>
-      {!error && renderIcon}
-      {error && renderFallbackIcon}
-    </>
-  )
+  return RD.fold(() => <></>, renderFallbackIcon, renderFallbackIcon, renderIcon)(remoteIconImage)
```

Comment:
I would render a fallback icon on error only and for or `initial` and `loading` states `Styled.IconBG`

Reply:
@veado there is no initial state by default behaviour of `useRemoteImage` (initial value is `pending`). for loading state here is a fallback renderer just 'cause i did not have a loading icon which should be placed. 
anyway i will place here whatever icon you say. just replace a few symbols.
just say exactly which ones ?:)

Reply:
For loading state you can use<IconWrapper><Styled.IconBG /><IconWrapper>, where [Styled.IconBG is just a "circle"](https://github.com/thorchain/asgardex-electron/blob/e6461d10c36cd2f68693931f1db5abd735ed5fd5/src/components/uielements/assets/assetIcon/AssetIcon.style.ts#L30-L37) with a size of an icon.

Reply:
Or we could use[ a custom `Spin` provided by `antd`](https://ant.design/components/spin/#components-spin-demo-custom-indicator) - it's up to you ;)

![Peek 2020-07-11 19-15](https://user-images.githubusercontent.com/61792675/87229729-fd50a980-c3aa-11ea-997a-33e9a77d46e5.gif)